try_test.go

```diff
@@ -98,6 +111,33 @@ func Test(t *testing.T) {
 	}
 }
 
+func TestFrame(t *testing.T) {
+	defer try.Recover(func(err error, frame runtime.Frame) {
+		if frame.File != "x.go" {
+			t.Errorf("want File=x.go, got %q", frame.File)
+		}
+		if frame.Line != 4 {
+			t.Errorf("want Line=4, got %d", frame.Line)
+		}
+	})
+//line x.go:4
```

Comment:
TIL this special pragma.

Reply:
Isn't it hideous? :) And like a hydra it has sprouted many heads.